<launch>

<arg name="node_name" />
<arg name="from_a" />
<arg name="to_b" />
<arg name="x" />
<arg name="y" />
<arg name="qx" default="0" />
<arg name="qy" default="0" />
<arg name="qz" default="0" />
<arg name="qw" default="1" />
<node pkg="tf" type="static_transform_publisher" name="odom_tf_launcher" args="$(arg x) $(arg y) 0 $(arg qx) $(arg qy) $(arg qz) $(arg qw) $(arg from_a) $(arg to_b) 100" />

</launch>
